content/.template.config/template.json

```diff
@@ -237,7 +237,7 @@
       ],
       "args": {
         "executable": "dotnet",
-        "args": "format --severity warn",
+        "args": "format --severity warn backend",
```

Comment:
Does it still actually work? I recall `dotnet format` now requiring to tell it what to fix: `[whitespace|style|analyzers]`

Reply:
It does work and it does not require that, no command works as using all of those 3.